src/components/aktivitetskrav/vurdering/VurderAktivitetskravSkjema.tsx

```diff
@@ -16,8 +16,7 @@ export interface VurderAktivitetskravSkjemaProps {
 interface Props<SkjemaValues> extends VurderAktivitetskravSkjemaProps {
   title: string;
   subtitle?: ReactElement;
-  beskrivelse?: ReactElement;
-  arsakVelger?: ReactElement;
+  children?: ReactElement[];
```

Comment:
Dette synes jeg er fint! 👍 Det er kanskje mer `React`ete å legge inn komponentene som barn, i stedet for å sende de inn i props 🤔